pkg/controller/ptpoperatorconfig/ptpoperatorconfig_controller.go

```diff
@@ -201,6 +202,26 @@ func (r *ReconcilePtpOperatorConfig) syncLinuxptpDaemon(defaultCfg *ptpv1.PtpOpe
 	data.Data["Namespace"] = names.Namespace
 	data.Data["ReleaseVersion"] = os.Getenv("RELEASEVERSION")
 	data.Data["KubeRbacProxy"] = os.Getenv("KUBE_RBAC_PROXY_IMAGE")
+	data.Data["SideCar"] = os.Getenv("SIDECAR_EVENT_IMAGE")
+	// configure EventConfig
+	data.Data["EnableEventPublisher"] = "false"
+	data.Data["EventTransportHost"] = defaultCfg.Spec.EventConfig.TransportHost
+	data.Data["HoldOverTimeout"] = defaultCfg.Spec.EventConfig.HoldOverTimeout
+	data.Data["MaxOffsetThreshold"] = defaultCfg.Spec.EventConfig.MaxOffsetThreshold
+	data.Data["MinOffsetThreshold"] = defaultCfg.Spec.EventConfig.MinOffsetThreshold
+	if defaultCfg.Spec.EventConfig.EnableEventPublisher {
+		data.Data["EnableEventPublisher"] = "true"
+		if defaultCfg.Spec.EventConfig.HoldOverTimeout == 0 {
+			data.Data["HoldOverTimeout"] = 5 //min 5 secs
```

Comment:
lets have a info log pointing out what is the time we use and why(no config provided)

Reply:
I can add those data to CR status .So you always know the config status. Will that work?

Reply:
I made this data available via CLI 

$oc get ptpoperatorconfig
```
NAME      EVENT ENABLED   HOLDOVER TIMEOUT   MAX THRESHOLD   MIN THRESHOLD   EVENT TRANSPORT HOST
default   false           5                  100             -100            
```